.gitignore

```diff
@@ -9,4 +9,6 @@ index.js
 /client
 /utils
 d.ts
-js.map
\ No newline at end of file
+js.map
+/server
+/lib
```

Comment:
se jogar em lib, vai quebrar o `package.json`
Nao sei como arrumar =.

Reply:
Vou alterar, depois estudo se é possível e se for implemento aqui!